src/components/logo/Logo.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+import Logo from './Logo';
+import notes from './Logo.stories.md';
+
+export const regular = () => <Logo />;
```

Comment:
The Styleguidist example sets the `title` attribute to "Box" -- can you include that here as well?